backend/app/api/auth.py

```
@@ -0,0 +1,151 @@
+        return Token(
+            access_token=access_token,
+            refresh_token=refresh_token,
+        )
+    except Exception as e:
+        raise HTTPException(status_code=400, detail=str(e))
+
+
+@router.post("/wallet")
+async def wallet_auth(
+    wallet_data: WalletAuth,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)